### PR TITLE
Add update and cancel events to subscriptions

### PR DESCRIPTION
**Pull request by danielfriis**

### CODE REVIEW

File: app/models/subscription.rb [hunk 1]
```diff
@@ -3,4 +3,8 @@ class Subscription < ApplicationRecord
   belongs_to :plan
 
   delegate :name, to: :plan, prefix: true
+
+  scope :active, -> { where("next_bill_date >= ? AND status = 'active'", Time.zone.now.to_date) }
```
Comment (kaspernj) [inline app/models/subscription.rb]: You should add table names, so the scopes can support other tables with similar columns names. `status` for example.

File: app/views/subscription_details/index.html.haml [hunk 2]
```diff
@@ -10,10 +10,11 @@
             %dt= Plan.model_name.human
             %dd= subscription.plan_name
             %dt= Subscription.human_attribute_name(:status)
-            %dd= subscription.status
+            %dd= subscription.status.capitalize
```
Comment (kaspernj) [inline app/views/subscription_details/index.html.haml]: I would argue you should handle translations here.

File: spec/factories/subscriptions.rb [hunk 3]
```diff
@@ -2,5 +2,7 @@
   factory :subscription do
     account
     plan
+    next_bill_date Date.tomorrow
+    status "active"
```
Comment (kaspernj) [inline spec/factories/subscriptions.rb]: It seems like `state_machine` would have been excellent for this.

File: app/models/subscription.rb (hunk 1 above)
Comment (danielfriis) [inline app/models/subscription.rb]: Ah, so `subscriptions.status` for instance?

File: spec/factories/subscriptions.rb (hunk 3 above)
Comment (danielfriis) [inline spec/factories/subscriptions.rb]: 👍

**Pull request merged**